.vscode/launch.json

```diff
@@ -198,6 +198,18 @@
         "shell", "ssh", "user@18.215.249.49",
       ],
     },
+    {
+      "name": "scan github org",
```

Comment:
did you mean to keep this in here

Reply:
Yes. I think it is useful for folks testing, specially now that we have https://github.com/hit-training.

I can remove it.